modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/PullRequest.java

```diff
@@ -91,8 +91,12 @@ public String getLabelsURL() {
 		return StringUtils.replace(labelsURL, "{/name}", "");
 	}
 
-	public String getOwnerUserName() {
-		return _ownerUserName;
+	public String getNumber() {
+		return Integer.toString(_number);
```

Comment:
Use `String.valueOf()` instead of `Integer.toString()`. BChan prefers this because you can use String.valueOf() for many different types without having to load a class for each type you need.